pgmpy/models/DynamicBayesianNetwork.py

```diff
@@ -0,0 +1,500 @@
+
+        +------+-----------------------+---------------------+
+        |diff: |          easy         |         hard        |
+        +------+------+------+---------+------+------+-------+
+        |intel:| dumb |  avg |  smart  | dumb | avg  | smart |
+        +------+------+------+---------+------+------+-------+
+        |gradeA| 0.1  | 0.1  |   0.1   |  0.1 |  0.1 |   0.1 |
+        +------+------+------+---------+------+------+-------+
+        |gradeB| 0.1  | 0.1  |   0.1   |  0.1 |  0.1 |   0.1 |
+        +------+------+------+---------+------+------+-------+
+        |gradeC| 0.8  | 0.8  |   0.8   |  0.8 |  0.8 |   0.8 |
+        +------+------+------+---------+------+------+-------+
+        """
+        for cpd in cpds:
+            if not isinstance(cpd, (TabularCPD, TreeCPD, RuleCPD)):
+                raise ValueError('Only TabularCPD, TreeCPD or RuleCPD can be'
+                                 ' added.')
+
+            if (cpd.variable, timeslice) not in self.dbn_cpds:
+                self.dbn_cpds[(cpd.variable, timeslice)] = cpd
+
+    def get_dbn_cpds(self, node=None, timeslice=None):
+        """
+        Returns the cpds that have been added till now to the graph
+        
+        Parameters
+        ---------
+        node: any hashable python object (optional)
+            The node whose CPD we want. If node not specified returns all the
+            CPDs added to the model.
+    
+        timeslice: integer value.
+                   should be ranging from 0 to the number of the
+                   slices present in the bayesian network.
+    
+        EXAMPLE
+        --------
+        >>> from pgmpy.models import DynamicBayesianNetwork as DBN
+        >>> from pgmpy.factors import TabularCPD
+        >>> student = DBN('D','G','I','S','L')
+        >>> grade_cpd = TabularCPD('G', 3, [[0.3, 0.05, 0.9, 0.5],[0.4, 0.25, 0.08, 0.3],[0.3, 0.7, 0.02, 0.2]], ['I','D'],[2,2])
+        >>> student.add_dbn_cpds(0, grade_cpd)
+        >>> student.get_dbn_cpds('G',0)
+        ╒═════╤═════╤══════╤══════╤═════╕
+        │ D   │ D_0 │ D_0  │ D_1  │ D_1 │
+        ├─────┼─────┼──────┼──────┼─────┤
+        │ I   │ I_0 │ I_1  │ I_0  │ I_1 │
+        ├─────┼─────┼──────┼──────┼─────┤
+        │ G_0 │ 0.3 │ 0.05 │ 0.9  │ 0.5 │
+        ├─────┼─────┼──────┼──────┼─────┤
+        │ G_1 │ 0.4 │ 0.25 │ 0.08 │ 0.3 │
+        ├─────┼─────┼──────┼──────┼─────┤
+        │ G_2 │ 0.3 │ 0.7  │ 0.02 │ 0.2 │
+        ╘═════╧═════╧══════╧══════╧═════╛
+        """
+        if node:
+            if timeslice:
+                if (node,timeslice) in self.dbn_cpds:
+                    return self.dbn_cpds[(node, timeslice)]
+            else:
+                list_of_cpds = []
+                for time_state in range(self.timestate + 1):
+                    list_of_cpds.append(self.dbn_cpds[(node, time_state)])
+                return list_of_cpds
+        else:
+            return self.dbn_cpds
+
+    def compute_initial_state(self):
```

Comment:
The method name is also a misnomer. ```compute_initial_state``` gives us a sense that it returns something, whereas it  doesn't. Rename it to something like ```initialize_initial_state``` or something like this.